tests/jquery.flot.touch.Test.js

```diff
@@ -28,6 +28,36 @@ describe("flot touch plugin", function () {
         expect(spy).toHaveBeenCalledWith('touchend', jasmine.any(Function));
     });
 
+    it('do not stop origin touch event propagation if it is allowed', () => {	
+        jasmine.clock().install().mockDate();	
+
+        var oldPropagateOriginTouch = options.propagateOriginTouch ;	
+        options.propagateOriginTouch = true;	
+
+        plot = $.plot(placeholder, [[]], options);	
+        var eventHolder = plot.getEventHolder(),	
+            spy = jasmine.createSpy('origin touch event handler');	
+
+        eventHolder.parentNode.addEventListener('touchstart', spy, { once: true });	
+        eventHolder.parentNode.addEventListener('touchmove', spy, { once: true });	
+        eventHolder.parentNode.addEventListener('touchend', spy, { once: true });
+
+        var bubbleTouchEvents = [
+            new UIEvent('touchstart', { bubbles: true }),
+            new UIEvent('touchmove', { bubbles: true }),
+            new UIEvent('touchend', { bubbles: true }),
+        ];
```

Comment:
The `simulate` from `webcharts-development-settings` cannot dispatch touch events with bubbles: true. So I use the low-level API for this test.